### ISSUE TITLE
Lot of Java Crash (One a day) on a quite huge server

### ISSUE THREAD

**Issue by Fuizziy** [TorchSpigot/Torch#139]
Hi there,
I've been using torch for a long time, and, honestly, this is the greatest thing that I've ever seen. However, these times, a lot of hard crash (which create hs_err_pid<pid> files at the root of the server) are happening. I don't know what causes these errors, but I know that removing torch helps me to avoid crashes. 
Is there something to do to prevent the server from crashing before the next stable update of torch ? As I can't build the latest torch versions, I don't know if it is already planned to patch it.

Thanks a lot

Here is one of the error

#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007f134cb551f4, pid=22297, tid=0x00007f1338849700
#
# JRE version: Java(TM) SE Runtime Environment (8.0_144-b01) (build 1.8.0_144-b01)
# Java VM: Java HotSpot(TM) 64-Bit Server VM (25.144-b01 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# J 7725 C2 net.minecraft.server.v1_11_R1.NBTTagCompound.getByte(Ljava/lang/String;)B (29 bytes) @ 0x00007f134cb551f4 [0x00007f134cb55040+0x1b4]
#
# Failed to write core dump. Core dumps have been disabled. To enable core dumping, try "ulimit -c unlimited" before starting Java again
#
# If you would like to submit a bug report, please visit:
#   http://bugreport.java.com/bugreport/crash.jsp
#

**Comment by Vlvxingze** [comment]
well,I get it.
But it.s.strange.
usually,java crash have no relate with torch.

sorry,I am not good at English

**Comment by Fuizziy** [comment]
So what do you mean by "I get it" ? Is it related to torch or not ? =) 
If no, what do you think I'd have to do ? If yes, do you know how to patch this ? :)

Thanks again for your amazing work !

**Comment by Fuizziy** [comment]
Up é.é

**Comment by Fuizziy** [comment]
So here are two beginnings of different errors

[https://pastebin.com/ZD0ffTTf](https://pastebin.com/ZD0ffTTf)
[https://pastebin.com/STqGHas3](https://pastebin.com/STqGHas3)

**Comment by Vlvxingze** [comment]
Well, I'm sorry I had to close my computer before I was sorry
I'm back!
I mean, in theory, JAVA crash doesn't have anything to do with servers,.
I'll try to fix it, and it probably won't fix, because I'm not sure it's the server's fault

**Comment by krusic22** [comment]
Just a corrupt chunk or entity.
